edenia/antelope-tools
# Feat/empty state 1066

### Show empty state when bp has no bp json

### What does this PR do?

- Resolve #1066 
- Fix : avoid to save health status when bp json is empty

### Steps to test

1. Run the project locally.
1. Go to block producers page.
1. Check the bps without bp json.

### Screenshot

#### Desktop
![imagen](https://user-images.githubusercontent.com/66583677/206753817-05725aa7-c903-479a-a602-fce5380cc2ae.png)

#### Mobile
![imagen](https://user-images.githubusercontent.com/66583677/206753703-d5817b6c-b499-4765-a5f6-d0a9f10b971a.png)
